network/discovery.go

```diff
@@ -206,19 +224,22 @@ func (d *discovery) findPeersCall(peerID peer.ID) ([]*peer.AddrInfo, error) {
 	if err != nil {
 		return nil, err
 	}
+
 	clt := proto.NewDiscoveryClient(stream.(*rawGrpc.ClientConn))
 
 	resp, err := clt.FindPeers(context.Background(), &proto.FindPeersReq{Count: 16})
 	if err != nil {
 		return nil, err
 	}
 
-	var addrInfo []*peer.AddrInfo
+	addrInfo := make([]*peer.AddrInfo, 0)
```

Comment:
Why don't you give size or capacity from len(resp.Nodes)?

Reply:
Fixed 👍 

db95f0f